lib/tether/ops_linux.go

```diff
@@ -960,14 +960,33 @@ func (t *BaseOperations) MountTarget(ctx context.Context, source url.URL, target
 	}
 	rawSource.WriteString(source.Path)
 
-	// NOTE: by default we are supporting "NOATIME" and it can be configurable later. this must be specfied as a flag.
-	// Additionally, we must parse out the "ro" option and supply it as a flag as well for this flavor of the mount call.
-	if err := Sys.Syscall.Mount(rawSource.String(), target, nfsFileSystemType, syscall.MS_NOATIME, mountOptions); err != nil {
+	if ip := net.ParseIP(source.Hostname()); ip != nil {
+		// NOTE: by default we are supporting "NOATIME" and it can be configurable later. this must be specfied as a flag.
+		// Additionally, we must parse out the "ro" option and supply it as a flag as well for this flavor of the mount call.
+		if err := Sys.Syscall.Mount(rawSource.String(), target, nfsFileSystemType, syscall.MS_NOATIME, mountOptions); err != nil {
+			log.Errorf("mounting %s on %s failed: %s", source.String(), target, err)
+			return err
+		}
+		return nil
+	}
+
+	log.Debugf("Looking up host %s", source.Hostname())
+	ips, err := net.LookupIP(source.Hostname())
+	if err != nil {
 		log.Errorf("mounting %s on %s failed: %s", source.String(), target, err)
```

Comment:
Could we update this error message to specify looking up the IP for the specified hostname failed? It seems like that might be useful context to provide to aid in debugging, as that might lead to different troubleshooting steps than focusing on the mounting process itself.

Reply:
I've thought that the variable 'err' contains corresponding looking-up failure info